pymcon/README.md

```diff
@@ -0,0 +1,62 @@
+# PyMCon Branding
```

Comment:
Add a link to the assets in the pymcon_website_2022 folder to ensure folks can find everything

Reply:
Fixed both!